colibri/likelihood.py

```diff
@@ -119,6 +124,13 @@ def log_likelihood(
             jax array with the value of the log-likelihood.
         """
         predictions, pdf = self.pred_and_pdf(params, fast_kernel_arrays)
+        predictions = predictions[self.central_values_idx]
+
```

Comment:
Is this actually needed? (probably by tr_idx and vl_idx?)
If so it would be good to add a comment